Source Clickhouse-strict-encrypt: failed ClickHouseStrictEncryptJdbcSourceAcceptanceTest

<!-- 
Welcome to Airbyte!
We're really appreciate your report and know that this will help us build an 
amazing tool. If you want to contribute yourself, you can find a good place 
to start by searching for the good-first-issues label or maybe... by trying 
to solve this one? (we can help debug this with you!)

Right now we are in alpha, so we're releasing versions a lot more frequently than 
normal. You can help us get to the root of the problem faster by filling out the
questionnaire below!

It's really important having all information and context. 
You can remove the examples bellow and fill out with your information.

--> 
## Current Behavior
`testSpec() `fails for source Clickhouse-strict-encrypt:



## Logs


<details>
<summary>LOG</summary> 

```

ClickHouseStrictEncryptJdbcSourceAcceptanceTest > testSpec() FAILED
    java.lang.AssertionError: expected:<io.airbyte.protocol.models.ConnectorSpecification@2e66340[documentationUrl=https://docs.airbyte.io/integrations/destinations/clickhouse,changelogUrl=<null>,connectionSpecification=***"$schema":"http://json-schema.org/draft-07/schema#","title":"ClickHouse Source Spec","type":"object","required":["host","port","database","username"],"additionalProperties":false,"properties":***"host":***"description":"Host Endpoint of the Clickhouse Cluster","type":"string"***,"port":***"description":"Port of the database.","type":"integer","minimum":0,"maximum":65536,"default":8123,"examples":["8123"]***,"database":***"description":"Name of the database.","type":"string","examples":["default"]***,"username":***"description":"Username to use to access the database.","type":"string"***,"password":***"description":"Password associated with the username.","type":"string","airbyte_secret":true***,"tunnel_method":***"type":"object","title":"SSH Tunnel Method","description":"Whether to initiate an SSH tunnel before connecting to the database, and if so, which kind of authentication to use.","oneOf":[***"title":"No Tunnel","required":["tunnel_method"],"properties":***"tunnel_method":***"description":"No ssh tunnel needed to connect to database","type":"string","const":"NO_TUNNEL","order":0***,***"title":"SSH Key Authentication","required":["tunnel_method","tunnel_host","tunnel_port","tunnel_user","ssh_key"],"properties":***"tunnel_method":***"description":"Connect through a jump server tunnel host using username and ssh key","type":"string","const":"SSH_KEY_AUTH","order":0***,"tunnel_host":***"title":"SSH Tunnel Jump Server Host","description":"Hostname of the jump server host that allows inbound ssh tunnel.","type":"string","order":1***,"tunnel_port":***"title":"SSH Connection Port","description":"Port on the proxy/jump server that accepts inbound ssh connections.","type":"integer","minimum":0,"maximum":65536,"default":22,"examples":["22"],"order":2***,"tunnel_user":***"title":"SSH Login Username","description":"OS-level username for logging into the jump server host.","type":"string","order":3***,"ssh_key":***"title":"SSH Private Key","description":"OS-level user account ssh key credentials in RSA PEM format ( created with ssh-keygen -t rsa -m PEM -f myuser_rsa )","type":"string","airbyte_secret":true,"multiline":true,"order":4***,***"title":"Password Authentication","required":["tunnel_method","tunnel_host","tunnel_port","tunnel_user","tunnel_user_password"],"properties":***"tunnel_method":***"description":"Connect through a jump server tunnel host using username and password authentication","type":"string","const":"SSH_PASSWORD_AUTH","order":0***,"tunnel_host":***"title":"SSH Tunnel Jump Server Host","description":"Hostname of the jump server host that allows inbound ssh tunnel.","type":"string","order":1***,"tunnel_port":***"title":"SSH Connection Port","description":"Port on the proxy/jump server that accepts inbound ssh connections.","type":"integer","minimum":0,"maximum":65536,"default":22,"examples":["22"],"order":2***,"tunnel_user":***"title":"SSH Login Username","description":"OS-level username for logging into the jump server host","type":"string","order":3***,"tunnel_user_password":***"title":"Password","description":"OS-level password for logging into the jump server host","type":"string","airbyte_secret":true,"order":4***]***,supportsIncremental=<null>,supportsNormalization=false,supportsDBT=false,supportedDestinationSyncModes=[],authSpecification=<null>,advancedAuth=<null>,additionalProperties=***]> but was:<io.airbyte.protocol.models.ConnectorSpecification@6671ec3b[documentationUrl=https://docs.airbyte.io/integrations/destinations/clickhouse,changelogUrl=<null>,connectionSpecification=***"$schema":"http://json-schema.org/draft-07/schema#","title":"ClickHouse Source Spec","type":"object","required":["host","port","database","username"],"additionalProperties":false,"properties":***"host":***"description":"The host endpoint of the Clickhouse cluster.","title":"Host","type":"string"***,"port":***"description":"The port of the database.","title":"Port","type":"integer","minimum":0,"maximum":65536,"default":8123,"examples":["8123"]***,"database":***"description":"The name of the database.","title":"Database","type":"string","examples":["default"]***,"username":***"description":"The username which is used to access the database.","title":"Username","type":"string"***,"password":***"description":"The password associated with this username.","title":"Password","type":"string","airbyte_secret":true***,"tunnel_method":***"type":"object","title":"SSH Tunnel Method","description":"Whether to initiate an SSH tunnel before connecting to the database, and if so, which kind of authentication to use.","oneOf":[***"title":"No Tunnel","required":["tunnel_method"],"properties":***"tunnel_method":***"description":"No ssh tunnel needed to connect to database","type":"string","const":"NO_TUNNEL","order":0***,***"title":"SSH Key Authentication","required":["tunnel_method","tunnel_host","tunnel_port","tunnel_user","ssh_key"],"properties":***"tunnel_method":***"description":"Connect through a jump server tunnel host using username and ssh key","type":"string","const":"SSH_KEY_AUTH","order":0***,"tunnel_host":***"title":"SSH Tunnel Jump Server Host","description":"Hostname of the jump server host that allows inbound ssh tunnel.","type":"string","order":1***,"tunnel_port":***"title":"SSH Connection Port","description":"Port on the proxy/jump server that accepts inbound ssh connections.","type":"integer","minimum":0,"maximum":65536,"default":22,"examples":["22"],"order":2***,"tunnel_user":***"title":"SSH Login Username","description":"OS-level username for logging into the jump server host.","type":"string","order":3***,"ssh_key":***"title":"SSH Private Key","description":"OS-level user account ssh key credentials in RSA PEM format ( created with ssh-keygen -t rsa -m PEM -f myuser_rsa )","type":"string","airbyte_secret":true,"multiline":true,"order":4***,***"title":"Password Authentication","required":["tunnel_method","tunnel_host","tunnel_port","tunnel_user","tunnel_user_password"],"properties":***"tunnel_method":***"description":"Connect through a jump server tunnel host using username and password authentication","type":"string","const":"SSH_PASSWORD_AUTH","order":0***,"tunnel_host":***"title":"SSH Tunnel Jump Server Host","description":"Hostname of the jump server host that allows inbound ssh tunnel.","type":"string","order":1***,"tunnel_port":***"title":"SSH Connection Port","description":"Port on the proxy/jump server that accepts inbound ssh connections.","type":"integer","minimum":0,"maximum":65536,"default":22,"examples":["22"],"order":2***,"tunnel_user":***"title":"SSH Login Username","description":"OS-level username for logging into the jump server host","type":"string","order":3***,"tunnel_user_password":***"title":"Password","description":"OS-level password for logging into the jump server host","type":"string","airbyte_secret":true,"order":4***]***,supportsIncremental=<null>,supportsNormalization=false,supportsDBT=false,supportedDestinationSyncModes=[],authSpecification=<null>,advancedAuth=<null>,additionalProperties=***]>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at io.airbyte.integrations.io.airbyte.integration_tests.sources.ClickHouseStrictEncryptJdbcSourceAcceptanceTest.testSpec(ClickHouseStrictEncryptJdbcSourceAcceptanceTest.java:145)


```

</details>

## Steps to Reproduce
1. Run ClickHouseStrictEncryptJdbcSourceAcceptanceTest

## Are you willing to submit a PR?
<!--- 
We accept contributions! 
Don't feel pressured, but if you want to contribute we can help you by giving some tips, 
highlighting the necessary code change or explaining any relevant point your feature will impact.
You can also send questions on #dev Slack channel.

We understand if you can't submit a PR and we're tremendously grateful
that you've already contributed by suggesting a new feature.
-->
Remove this with your answer.

--- a/airbyte-integrations/connectors/source-clickhouse-strict-encrypt/Dockerfile
+++ b/airbyte-integrations/connectors/source-clickhouse-strict-encrypt/Dockerfile
@@ -16,5 +16,5 @@ ENV APPLICATION source-clickhouse-strict-encrypt
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.1.2
+LABEL io.airbyte.version=0.1.3
 LABEL io.airbyte.name=airbyte/source-clickhouse-strict-encrypt

--- a/airbyte-integrations/connectors/source-clickhouse-strict-encrypt/src/test-integration/resources/expected_spec.json
+++ b/airbyte-integrations/connectors/source-clickhouse-strict-encrypt/src/test-integration/resources/expected_spec.json
@@ -8,11 +8,13 @@
     "additionalProperties": false,
     "properties": {
       "host": {
-        "description": "Host Endpoint of the Clickhouse Cluster",
+        "description": "The host endpoint of the Clickhouse cluster.",
+        "title": "Host",
         "type": "string"
       },
       "port": {
-        "description": "Port of the database.",
+        "description": "The port of the database.",
+        "title": "Port",
         "type": "integer",
         "minimum": 0,
         "maximum": 65536,
@@ -20,16 +22,19 @@
         "examples": ["8123"]
       },
       "database": {
-        "description": "Name of the database.",
+        "description": "The name of the database.",
+        "title": "Database",
         "type": "string",
         "examples": ["default"]
       },
       "username": {
-        "description": "Username to use to access the database.",
+        "description": "The username which is used to access the database.",
+        "title": "Username",
         "type": "string"
       },
       "password": {
-        "description": "Password associated with the username.",
+        "description": "The password associated with this username.",
+        "title": "Password",
         "type": "string",
         "airbyte_secret": true
       }

--- a/docs/integrations/sources/clickhouse.md
+++ b/docs/integrations/sources/clickhouse.md
@@ -90,6 +90,7 @@ Using this feature requires additional configuration, when creating the source. 
 
 | Version | Date | Pull Request                                             | Subject                                                                    |
 |:---| :---  |:---------------------------------------------------------|:---------------------------------------------------------------------------|
+| 0.1.3 | 2021-12-29 | [\#9182](https://github.com/airbytehq/airbyte/pull/9182) [\#8958](https://github.com/airbytehq/airbyte/pull/8958) | Add support for JdbcType.ARRAY. Fixed tests                                |
 | 0.1.2 | 2021-12-01 | [\#8371](https://github.com/airbytehq/airbyte/pull/8371) | Fixed incorrect handling "\n" in ssh key                                   |
 | 0.1.1 | 20.10.2021 | [\#7327](https://github.com/airbytehq/airbyte/pull/7327) | Added support for connection via SSH tunnel(aka Bastion server).           |
 | 0.1.0 | 20.10.2021 | [\#7127](https://github.com/airbytehq/airbyte/pull/7127) | Added source-clickhouse-strict-encrypt that supports SSL connections only. |

